### PR TITLE
RD-2500 Remove jQuery dependency from External.ts.

### PR DESCRIPTION
**Pull request by stepek**

### CODE REVIEW

File: app/utils/External.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ import log from 'loglevel';
 import 'isomorphic-fetch';
 import { saveAs } from 'file-saver';
 import JSZip from 'jszip';
+import { stringify as queryQtringify } from 'query-string';
```
Comment (kubama) [inline app/utils/External.ts:8]: `qtringify`? Is it a typo 🙂?

File: app/utils/External.ts [hunk 2]
```diff
@@ -282,9 +284,7 @@ export default class External {
 
     // eslint-disable-next-line class-methods-use-this
     protected buildActualUrl(url: string, data?: Record<string, any>) {
-        // TODO: RD-258
-        // @ts-ignore Cannot find $
-        const queryString = data ? (url.indexOf('?') > 0 ? '&' : '?') + $.param(data, true) : '';
+        const queryString = data ? (url.indexOf('?') > 0 ? '&' : '?') + queryStringify(data) : '';
```
Comment (Gelio) [inline app/utils/External.ts:287]: Did you consider using https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams to build the query string?

This way we won't be pulling in third-party code :slightly_smiling_face:

File: test/jest/utils/External.test.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import External from 'utils/External';
+
+describe('(Utils) External', () => {
+    describe('buildActualUrl (protected)', () => {
+        const external = new External({});
+
+        it(`should return correct url with params`, () => {
```
Comment (Gelio) [inline test/jest/utils/External.test.ts:7]: Minor:

We could also add a test when the URL already contains a query string :slightly_smiling_face:

File: app/utils/External.ts (hunk 2 above)
Comment (Gelio) [reply app/utils/External.ts:287]: Actually, nevermind, as it won't handle arrays out of the box :smile: The test uses arrays, not sure if we use them in production, but w/e :smile: It's ok :slightly_smiling_face:

File: app/utils/External.ts [hunk 4]
```diff
@@ -5,6 +5,8 @@ import log from 'loglevel';
 import 'isomorphic-fetch';
 import { saveAs } from 'file-saver';
 import JSZip from 'jszip';
+import { stringify as queryStringify } from 'query-string';
```
Comment (Gelio) [inline app/utils/External.ts:8]: Minor:

I would call it

```suggestion
import { stringify as stringifyQueryString } from 'query-string';
```

as it sounds to me more like a function :slightly_smiling_face:

**Review by Gelio — DISMISSED** [review]

**Review by kubama — APPROVED** [review]

File: app/utils/External.ts (hunk 2 above)
Comment (stepek) [reply app/utils/External.ts:287]: 👍

File: test/jest/utils/External.test.ts (hunk 3 above)
Comment (stepek) [reply test/jest/utils/External.test.ts:7]: as you wish :)

**Review by Gelio — APPROVED** [review]

**Pull request merged**